bib/cmd/bootc-image-builder/main.go

```diff
@@ -174,11 +174,23 @@ func build(cmd *cobra.Command, args []string) {
 	check(saveManifest(mf, manifestPath))
 
 	fmt.Printf("Building %s\n", manifest_fname)
+	check(runOSBuild(mf, osbuildStore, outputDir, exports))
+	fmt.Printf("Build complete. Results saved in %s\n", outputDir)
+	return nil
```

Comment:
The `build` command is the value of the `RunE` field in the `cobra.Command` struct so it needs to return an error.  We can use the `Run` field though.
But what we should actually do, I think, is pass errors up when they're related to bad command line arguments, since returning an error from the `RunE` function of the command will print the usage, which is a good idea in such cases.